components/authn-service/tokens/pg/sql/12_add_filter_function_for_token_projects.up.sql

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
extra line? also, does that even matter?

Reply:
I've mechanically translated the inline statements in our go code into these files -- so, that one was probably like
```go
Exec(`
SOME SQL HERE
`
)
```
and that backtick right behind the `(` was what gave us this newline. 🤷‍♂ I think it doesn't matter 😃